code/modules/events/syndicate_retribution.dm

```diff
@@ -5,7 +5,9 @@
 	required_elapsed_round_time = 40 MINUTES
 	weight = 88
 
+#ifdef RP_MODE
 	disabled = 1
+#endif
```

Comment:
```suggestion
 	disabled = 1
```
Still disabled for now. No dev really wants to enable it currently. I do want to see the fixes in, though